pkg/security/ebpf/c/process.h

```diff
@@ -5,22 +5,22 @@
 #include <linux/sched.h>
 
 static struct proc_cache_t * __attribute__((always_inline)) fill_process_data(struct process_context_t *data) {
-    // Process data
-    struct task_struct *task = (struct task_struct *)bpf_get_current_task();
-
-    struct nsproxy *nsproxy;
-    bpf_probe_read(&nsproxy, sizeof(nsproxy), &task->nsproxy);
-
-    struct pid_namespace *pid_ns;
-    bpf_probe_read(&pid_ns, sizeof(pid_ns), &nsproxy->pid_ns_for_children);
-    bpf_probe_read(&data->pidns, sizeof(data->pidns), &pid_ns->ns.inum);
-
-    // TTY
-    struct signal_struct *signal;
-    bpf_probe_read(&signal, sizeof(signal), &task->signal);
-    struct tty_struct *tty;
-    bpf_probe_read(&tty, sizeof(tty), &signal->tty);
-    bpf_probe_read_str(data->tty_name, TTY_NAME_LEN, tty->name);
+//    // Process data
```

Comment:
It's probably better to just remove it. If we need it again, we'll use the git history

Reply:
Happy to remove it, but then shouldn't we remove all the fields in `process_context_t` and in the structure `ProcessEvent` that are related to the `tty`, `pidns`, and the fields we can't collect right now ?

Hopefully runtime compilation is right around the corner, it wouldn't stay commented for long 🤔 

Let me know what you think 😃